### ISSUE TITLE
Implement HALT bug?

### ISSUE THREAD

**Issue by brunodea** [brunodea/gebemula#2]
On page 20 of the GB manual one can read:

> The instruction immediately following the HALT instruction is "skipped" when interrupts are disabled (DI).

I believe that the emulator works well without implementing this. However, some games may take advatange of this bug, so it would be interesting to implement it for testing purposes.

**Comment by vitords** [comment]
Would be fun to test it out, but not a primary concern right now.

**Comment by brunodea** [comment]
Yeah, if we do this now we could forget about it and leave the bug there. So I'll just leave this issue here for reminding us in the future.
